src/crypto.cc

```diff
@@ -192,3 +209,66 @@ bool Crypto::parseP12(FILE *p12_fp, const std::string &p12_password, const std::
 
   return true;
 }
+
+bool Crypto::generateRSAKeyPair(const std::string &public_key, const std::string &private_key) {
+  int bits = 1024;
```

Comment:
Final version will use ED25519, but the server doesn't support that yet.  I agree that we should follow the UPTANE minimum key size though.